content/docs/tasks.md

```diff
@@ -0,0 +1,178 @@
+# Tasks
```

Comment:
or under developing operators? 🤔

Reply:
> I just feel like we have too many stuff on the top level

Tasks absolutely do not belong on the top-level. But I had to start somewhere 🤷‍♀ I think that all three: `Features`, `Developing Operators`, `Examples` and now `Tasks` belong under the same ☂️ Wdyt about:

```
└── Developing Operators
    ├── Introduction (former developing-operators.md)
    ├── Task (tasks.md)
    ├── Features (features.md)
    └── Examples (examples/README.md)
```

Reply:
I am not sure about features, because e.g. webhook is more about managing operators then developing them (and I put it under features). Maybe we should have the docs separated by personas actually and features are too generic.

So I think I like what you're proposing, we should have developing operators with tasks and then running operators with webhook? 🤔 

Btw. I am fine with doing this restructure in another PR - up to you :)

Reply:
Ah, webhooks PR hasn't been merged yet but you put it into features? 🤔 But I like the separation in "developing" and "managing". Though, current `feature.md` content `Controlled Parameter Changes` is more about...developing?

Reply:
yeah ... I don't know. Also where does cleanup plan go, that could be both... 🤔but probably also developing? but I think that pipes go definitely into developing...

Reply:
Ok, so I left `Features` at the top for now - not sure where it fits best. But the rest looks like the following:

```
Docs/
...
├── Developing Operators
│   ├── Concepts
│   ├── First Operator (former developing-operators.md)
│   └── Tasks
│   └── Examples
│       ├── Apache Flink
│       ├── Apache Kafka
│       └── Apache ZooKeeper
```
wdyt?

Reply:
I am not sure about the examples as the content of those pages right now seems to me more like operating, that developing

With concepts I am undecided ... right now they more describe the CRD structure, I am not sure you need that when developing actually, maybe more when operating? I don't know, that page is still ... meh ... to me :) but we could start with putting it there

First Operator I would rename to `Getting started`

But overall much better structure than the current one 👍 :)